Content.Shared/_Goobstation/Devil/DevilEvents.cs

```diff
@@ -0,0 +1,71 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Content.Shared._Goobstation.Devil.Contract;
+using Content.Shared.Inventory;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared._Goobstation.Devil;
```

Comment:
`Robust.Shared.Serialization` is unused
```suggestion
using Content.Shared.Inventory;

namespace Content.Shared._Goobstation.Devil;
```